CMakeLists.txt

```diff
@@ -317,6 +317,18 @@ add_subdirectory(misc)
 # property which is set implicitly for custom command outputs
 include(misc/generated/CMakeLists.txt)
 add_subdirectory(boards)
+
+FILE(GLOB_RECURSE EXT_MODULES ${CONTAINER_DIR}/modules/*/module.yml)
+FILE(GLOB_RECURSE EXT_HALS ${CONTAINER_DIR}/hals/*/module.yml)
+set(modules ${EXT_MODULES} ${EXT_HALS})
+foreach(module_file ${modules})
+  message(${module_file})
+  get_filename_component(module_dir ${module_file} DIRECTORY)
+  string(REPLACE ${CONTAINER_DIR}/ "" MODULE_BINARY_DIR ${module_dir})
+  #message(STATUS ${module_file} " " ${module_dir})
```

Comment:
I don't think we should have commented out code at all in a PR, regardless of RFC or not